home/models.py

```diff
@@ -219,3 +220,172 @@ class Footer(models.Model):
 
     def __str__(self):
         return self.title
+
+
+@register_setting
+class SiteSettings(BaseSetting):
```

Comment:
@dannyparsons Do we really need all of these settings? For example, where will the content rotation settings be used?

@sheralim012 I suggest, and if Danny agrees, that we cull this list of settings to ones which we are actively using right now in v2 and leave it to others to add settings here when features are implemented.

Reply:
@dannyparsons let me know which fields to keep and which fields to let go.

Reply:
I've asked a question on the issue about the need for content rotation. I think all the others have a clear use. I believe all the GA and FB analytics will still be used.

Reply:
Content rotation isn't in use in v1 so let's remove this.

Reply:
content rotation fields are removed.